test/blackbox-tests/test-cases/findlib-error/run.t

```diff
@@ -3,8 +3,7 @@ We are dropping support for findlib in dune
   $ dune build --root in-dune target.txt
   Entering directory 'in-dune'
   File "dune", line 2, characters 25-37:
-  Error: The findlib special variable is not supported in jbuild files, please use lib instead:
-  %{lib:pkg} in dune files
+  Error: Variable %{findlib:pkg} has been renamed to %{lib:pkg} since 1.0
```

Comment:
We should keep the term variable for variables without payload. These should be called forms, so:

```
Error: The %{findlib:...} form has been renamed to %{lib:...} since version 1.0 of the dune language.
```

Note: we need to make sure to not print the actual payload in such error messages. If we include the payload (`pkg` in thie case), then we are talking about this instance of the form. If we print `...`, we are talking about the form in general, which is what we want here.